frontend/public/components/dashboard/details-card/detail-item.tsx

```diff
@@ -3,17 +3,28 @@ import classNames from 'classnames';
 
 import { LoadingInline } from '../../utils';
 
-export const DetailItem: React.FC<DetailItemProps> = React.memo(({ title, value, isLoading }) => (
-  <React.Fragment>
-    <dt className="co-details-card__item-title">{title}</dt>
-    <dd className={classNames('co-details-card__item-value', {'text-secondary': !value})}>
-      {isLoading ? <LoadingInline /> : value || 'Unavailable'}
-    </dd>
-  </React.Fragment>
-));
+export const DetailItem: React.FC<DetailItemProps> = React.memo(
+  ({ title, isLoading, children, error = false }) => {
+    let status: React.ReactNode|string;
+    if (isLoading) {
+      status = <LoadingInline />;
+    } else if (error) {
+      status = <span className="text-secondary">Unavailable</span>;
+    } else {
+      status = children;
+    }
+    return (
+      <React.Fragment>
+        <dt className="co-details-card__item-title">{title}</dt>
+        <dd className={classNames('co-details-card__item-value', {'text-secondary': error})}>{status}</dd>
```

Comment:
no need for `classNames` as `text-seconday` is included in `error` definition already
```suggestion
        <dd className="co-details-card__item-value">{status}</dd>
```